packages/matter.js/src/tlv/TlvCodec.ts

```diff
@@ -339,6 +339,34 @@ export class TlvCodec {
         }
     }
 
+    public static getTagByteLength(tag?: TlvTag) {
```

Comment:
I can see the byte length stuff is implemented and tested but I can't find where it's used, is that a future commit or did I miss it?

Reply:
It is used in the method that puts  together the datareport and where the size is calculated.

Reply:
PS: aah ok this one is used in AnySchema ... and the one from AnySchema ist used by encoding logic